examples/simple-game-server/gameserverset.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usually you would define a Fleet rather than a GameServerSet
+# directly. This is here mostly for testing purposes
+
+apiVersion: "agones.dev/v1"
```

Comment:
Wondering if we should delete this file -- it actually serves no purpose other than potentially Agones developer testing.

Reply:
+1 to deleting.

Reply:
Makes sense